tasks/setup/CentOS_6.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Set variable with services List for CentOS 6
+  command: chkconfig --list  # noqa 303
+  register: __timesync_register_services
+  check_mode: no
+  changed_when: false
+
+- name: Set variable `timesync_services` with filtered uniq service names
+  set_fact:
+    timesync_services: |
+      {{ __timesync_register_services.stdout_lines |
+          map('regex_replace', '[.]service.*$', '') |
+          map('regex_replace', '@$', '') |
+          unique |
+          list }}
```

Comment:
`chkconfig --list` output looks like this:
```
# chkconfig --list
abrt-ccpp      	0:off	1:off	2:off	3:on	4:off	5:on	6:off
abrtd          	0:off	1:off	2:off	3:on	4:off	5:on	6:off
acpid          	0:off	1:off	2:on	3:on	4:on	5:on	6:off
anamon         	0:off	1:off	2:off	3:off	4:off	5:off	6:off
```
Note that there may be a combination of space and tab characters after the service name.  Try something like this to parse it into a list variable:
```

- name: Set variable `timesync_services` with filtered uniq service names
  set_fact:
    timesync_services: |
      {{ __timesync_register_services.stdout_lines |
          map('regex_replace', '[ \t].*$', '') |
          unique |
          list }}
```
that is - delete from the first space or tab to the end of the line.  This should give you a list like
abrt-ccpp
abrtd
acpid
anamon

Reply:
Guys, this will never will be ideal.
We should stop constant rebase like this. )

If you agree that we need this change please accept Pull Request.

I have checked `chkconfig --list` after service name it always return whitespace on CentOS 6 and CentOS 7.
I don't see requirement here to add tab handling.

Reply:
> Guys, this will never will be ideal.

But it should work and not cause errors.  The current commit https://github.com/linux-system-roles/timesync/pull/76/files#diff-d65d631b9c97cb4867ddb320fbf11cd7R11 will not work on EL6 and will cause errors.  So it must be fixed, one way or another.

> I have checked chkconfig --list after service name it always return whitespace on CentOS 6 and CentOS 7.

I see tabs on some of the services in my output.  Which is why I suggested to add the tabs.  Is it a problem to have a regex that looks for either spaces or tabs?

I appreciate the fact that this PR has taken an inordinately long time to review, and a lot of review comments and change requests (now 1 week and many changes requested), and I thank you for your patience.  I hope you appreciate that we try to have high standards for the benefit of our customers.

Reply:
Hi Richard
Now I understand.
Didn't say that in some cases there are problems on CentOS 6 with regex. Replaced regex.